examples/apollo-service-example/src/main/java/com/spotify/zoltar/examples/apollo/ServiceRunner.java

```diff
@@ -61,7 +62,8 @@ static void configure(final Environment environment) {
     final URI modelPath = URI.create(config.getString("iris.model"));
     final URI settingsPath = URI.create(config.getString("iris.settings"));
     final SemanticMetricRegistry metricRegistry = environment.resolve(SemanticMetricRegistry.class);
-    final PredictorMetrics metrics = SemanticPredictorMetrics.create(metricRegistry);
+    final PredictorMetrics metrics =
+        SemanticPredictorMetrics.create(metricRegistry, MetricId.EMPTY);
```

Comment:
What's the difference between using `MetricId.EMPTY` vs `MetricId.build()` (which was used before)?  Did build create some random UUID tag or something? And the idea now is we tag first using model id, and then add another tag for the metric type?

Reply:
The idea here is to allow users to add custom tag's to it i.e add service name tag. Model id and other tags that we create internally will be appended.

there's no difference between `MetricId.EMPTY` and `MetricId.build()` and it doesn't create random tags.